infrastructure/variables.tf

```diff
@@ -4,15 +4,21 @@ variable "product" {
 
 variable "component" {
   type = string
+  default="postgres-db-v11"
 }
 
+variable "component-V14" {
+  type = string
+  default="postgres-db-v14"
+}
 variable "location" {
   type = string
   default = "UK South"
 }
 
 variable "env" {
   type = string
+  default="demo"
```

Comment:
why it is defined here?

Reply:
i thought for this demo branch i will default it  but i can keep it a s a variable